styles.css

```diff
@@ -107,3 +107,450 @@ a {
+}
+
+.card {
+  display: grid;
+  flex-direction: column;
+  align-items: center;
+  padding: 0.2rem 0.1rem;
+  gap: 1.5rem;
+}
+
+.card-enable {
+  width: 327px;
+  height: 474px;
+  display: grid;
+}
+
+.img-placeholder {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+  justify-content: flex-end;
+  background: #ebf0ee;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+}
+
+.project-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 1rem;
+  gap: 1rem;
+  background: #fff;
+  border-radius: 0.5rem;
+}
+
+.project-text ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.project-text li {
+  margin: 0;
+  padding: 0;
+}
+
+.project-title {
+  font-weight: 600;
+  font-size: 1.5rem;
+  line-height: 2em;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #3a4a42;
+  margin: 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.5rem 0.7rem;
+  background: #ebf0ee;
+  border-radius: 0.2rem;
+}
+
+.tag-text {
+  font-weight: 600;
+  font-size: 0.7rem;
+  line-height: 1em;
+  letter-spacing: 0.03em;
+  color: #3a4a42;
+  margin: 0.5rem;
+}
+
+.see-project {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  padding-top: 1.8rem;
+}
+
+.btn-project {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.7rem;
+  gap: 0.6rem;
+  border: none;
+  background: #36b37f;
+  border-radius: 0.2rem;
+}
+
+.see-project-text {
+  font-weight: 500;
+  font-size: 1.6rem;
+  line-height: 1.5em;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 4rem 1.5rem;
+  gap: 3.6rem;
+  background: url("Assests/Images/about.svg");
+  background-repeat: no-repeat;
+  margin: 0;
+}
+
+.about-me-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  gap: 1rem;
+  width: 351px;
+  height: 333px;
+  margin: 0;
+}
+
+.about-me-title {
+  font-weight: 700;
+  font-size: 2.5rem;
+  line-height: 3.2rem;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #172b4d;
+  margin: 0;
+}
+
+.about-me-content {
+  font-weight: 400;
+  font-size: 1.2rem;
+  line-height: 2rem;
+  text-align: center;
+  color: #42526e;
+  margin: 0;
+}
+
+.btn-resume {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0.7rem;
+  gap: 0.6rem;
+  width: 159px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 0.2rem;
+  border: none;
+  margin-top: 1rem;
+}
+
+.get-my-resume {
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
```

Comment:
- [ ] Kindly make sure the **About  me** section follows the design like the one on the Figma template(**e.g.: Add the full background image and position it perfectly, missing the `Get My Resume` button as it overlapped by technologies cards,...**). Make sure you check everything from the Figma template and apply it to your project.

| Figma Design | Your Design | 
| ------------------ | ------------------- |
| ![about_f3](https://user-images.githubusercontent.com/83042742/185718953-ca12e4cb-9869-47c9-9084-1a47fbf58432.png) | ![Screenshot from 2022-08-20 01-27-45](https://user-images.githubusercontent.com/83042742/185719076-06a036db-4d23-4e6c-8b2e-0ab27e25a52d.png) |